src/orderly.py

```diff
@@ -34,12 +34,19 @@ def configure_orderly(service, initialise_volume):
 def configure_orderly_envir(service):
     password_group = service.settings['password_group']
     api_server = service.settings['instance_name'].lower()
+    # These are the entries in montagu-reports/orderly_config.yml:
+    # update here if more are added.
+    known = ["science", "uat", "production"]
+    if api_server in known:
+        slack_url = '"{}"'.format(
+            get_secret("slack/orderly-webhook", field="url"))
+    else:
+        api_server = "~"
+        slack_url = "~"
+
     if api_server == "(unknown)" or api_server == "teamcity":
```

Comment:
Shouldn't need this bit anymore, if we've already set to ~ for anything other than known?

Reply:
thanks, totally should have been removed

Reply:
removed now